src/main/java/com/sap/sgs/phosphor/fosstars/data/github/LocalRepository.java

```diff
@@ -225,6 +225,23 @@ public void pull() throws IOException {
     return Optional.of(Files.newInputStream(path));
   }
 
+  /**
+   * Read lines in a file.
+   *
+   * @param file A path to the file.
+   * @return A list of lines in the file if it exists.
+   * @throws IOException If something went wrong.
+   */
+  public Optional<List<String>> readLinesOf(Path file) throws IOException {
+    Objects.requireNonNull(file, "On no! File name is null!");
+    Path path = info.path().resolve(file);
+    if (!Files.isRegularFile(path)) {
+      return Optional.empty();
+    }
+
+    return Optional.of(IOUtils.readLines(Files.newInputStream(path)));
```

Comment:
Please use `Files.newInputStream(path)` with `try-with-resources` block.

Reply:
Good catch. It's now fixed. I opened #318 to make sure that we fixed all such issues.